test/IntegrationTests/SmokeTests.cs

```diff
@@ -267,6 +308,58 @@ public void PrometheusExporter()
     }
 #endif
 
+#if NETFRAMEWORK
+    [Fact]
+    [Trait("Category", "EndToEnd")]
+    public void PrometheusAndOtlpMetricsExporter()
```

Comment:
@open-telemetry/dotnet-instrumentation-approvers, what do you think `PrometheusMetricsExporter` and `OtlpMetricsExporter` to reduce the execution of integration tests? Basically this test combines both tests.